postcss.config.js

```diff
@@ -13,19 +18,15 @@ module.exports = (ctx) => {
             require('cssnano')({}),
             require('postcss-url')([
                 {
-                    filter: /\.svg/,
+                    filter: /\.(svg|png|jpg|gif|eot|ttf|woff|woff2)$/,
                     basePath: path.resolve(__dirname, 'assets/'),
-                    url: 'inline',
```

Comment:
We still want some assets to be inlined, don't we? I don't know what `postcss-url` normally does in this case, are files that are considered "small enough" inlined?